### PR TITLE
helpers/SimplePieFileGuzzle: handle case where headers is given but is null.

### PR DESCRIPTION
**Pull request by desaintmartin**

### CODE REVIEW

File: helpers/SimplePieFileGuzzle.php [hunk 1]
```diff
@@ -6,10 +6,13 @@
  * Bridge to make SimplePie fetch resources using Guzzle library
  */
 class SimplePieFileGuzzle extends \SimplePie_File {
-    public function __construct($url, $timeout = 10, $redirects = 5, $headers = [], $useragent = null, $force_fsockopen = false) {
+    public function __construct($url, $timeout = 10, $redirects = 5, $headers = null, $useragent = null, $force_fsockopen = false) {
         $this->url = $url;
         $this->permanent_url = $url;
         $this->useragent = $useragent;
+        if ($headers == null) {
```
Comment (jtojnar) [inline helpers/SimplePieFileGuzzle.php]:
```suggestion
        if ($headers === null) {
```

**Comment by jtojnar** [comment]
For reference, this is where it is called with `null`:

https://github.com/simplepie/simplepie/blob/11ddaed291ccd1b2628eef433a6bef58a1e4f920/library/SimplePie/Locator.php#L411

**Comment by jtojnar** [comment]
Thanks for fixing this.

**Comment by jtojnar** [comment]
Hi @desaintmartin. Previously, it [was not clear](https://github.com/fossar/selfoss/issues/1218) if selfoss was licensed under GPL 3 only, or also any later version. Could you clarify whether you are fine with licensing your contribution under `GPL-3-or-later`?

Thanks again and sorry for the confusion.

**Comment by desaintmartin** [comment]
As someone who wrote gpl code for years for a living, I could not be more happy if selfoss is GPL-3-or-later! :)

**Pull request merged**